### PR TITLE
Starter kit Docs

### PR DESCRIPTION
**Pull request by mwakaba2**
## Todos

- [x] Mariko: Complete the "get started" page
- [x] Mariko: For each page, add a banner indicating the docs will get updated frequently with new bots in the coming months
- [x] Christian: For each starter kit, include all work in progress bots in the docs to show up that more bots are on its way. (instead of bot details, add a "coming soon!" message)
- [x] Christian: Include the [web3 kill chain diagrams](https://docs.google.com/presentation/d/1PIupEnrRonphIVJQjQbRaE5ZyZ6Nt2eq8a2VX1yS0Qc/edit#slide=id.g1258bcf447b_0_163) for each starter kit. 
- [x] Include the [Kill chain blog post](https://forta.org/blog/web3-kill-chain/) in the intro
- [ ] After merging docs, ask Ivan to add link to the docs in the Forta App Monitor Contracts Form

### CODE REVIEW

**Comment by mwakaba2** [comment]
hey @ivandiazwm, we're working on adding the starter kit docs in the Forta SDK, and we'd like to include a doc link to each starter kit in the Forta app monitor form so that users can learn more about the bots in each subscription groups.
Once a user selects a subscription type that's a starter kit, would it be possible to show a link to the following docs?

- Forta Curated NFT Security Bots --> https://docs.forta.network/en/latest/nft-starter-kit/
- Forta Curated Defi Security Bots --> https://docs.forta.network/en/latest/defi-starter-kit/
- Forta Curated Bridge Security Bots -->  https://docs.forta.network/en/latest/bridge-starter-kit/
- Forta Curated Stablecoin Security Bots --> https://docs.forta.network/en/latest/stablecoin-starter-kit/

The above doc pages will be available once we merge this PR.

**Comment by mwakaba2** [comment]
@haseebrabbani and @christian-forta I'd appreciate a review for the "Get Started" page and overall doc layout. Please feel free to make direct edits too.

File: docs/starter-get-started.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+2. Click on "Monitor my contracts" or "Monitor my wallet". If you clicked on "Monitor my contracts", select a threat detection kit.
+    <p align="center">
+        <img alt="Forta Threat Detection Kits" src="../threat-detection-kits.png">
+    </p>
+4. Sign in with your wallet to continue.
+    <p align="center">
+        <img width="300px" alt="Wallet Sign" src="../wallet-sign.png">
+    </p>
+5. Select or update the selected threat detection kit in the "Select Subscription type" dropdown.
+6. Add a contract/wallet address to watch.
```
Comment (christian-forta) [inline docs/starter-get-started.md:36]: I think here we need to highlight a bit more that subscribers need to create a set of addresses pertaining to their protocol. That includes contracts, but also protocol specific token addresses. Also, if this list is dynamic (e.g. new liq pool contracts being created on the fly) the list needs to be maintained.

Do you know whether the field already allows to take multiple addresses. @ivandiazwm ?

File: docs/starter-get-started.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Get Started
+
+## What is a threat detection kit?
+
+If you are new to Forta, a threat detection kit will help you get started on monitoring and protecting Web3 projects and assets right away with little to no technical and security experience.
```
Comment (christian-forta) [inline docs/starter-get-started.md:5]: I think no security expertise is a bit much. I would rephrase to 'no custom bot development needed to get security relevant value from Forta.' - security expertise to monitor these alerts is very much required.

File: docs/starter-get-started.md (hunk 1 above)
Comment (christian-forta) [reply docs/starter-get-started.md:36]: if it takes multiple addresses, we should change the bullet to 'add contract/wallet addresses separated by comma' or something like that

File: docs/starter-get-started.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+5. Select or update the selected threat detection kit in the "Select Subscription type" dropdown.
+6. Add a contract/wallet address to watch.
+7. Choose notification type and destination.
+<p align="center">
+    <img width="500px" alt="Alert Subscription Form" src="../alert-subscription-form.png">
+</p>
+
+## What are detection bot templates?
+
+Each threat detection kit may also include detection bot templates. The bot templates allow users to quickly customize, create, and deploy bots without developing one from scratch.
```
Comment (christian-forta) [inline docs/starter-get-started.md:44]: I'd add something like 'While the existing threat detection kit bots provide broad coverage for security relevant issues, sometimes the bot needs protocol specific information. Detection bot template allow users to create these custom bots quickly without the need to develop one from scratch.'

File: docs/starter-kit-bot-details.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
+# Starter Kit Bot Details
```
Comment (christian-forta) [inline docs/starter-kit-bot-details.md:1]: do the anchors work from the individual kit pages?

**Review by christian-forta — APPROVED** [review]

File: docs/starter-kit-bot-details.md (hunk 4 above)
Comment (mwakaba2) [reply docs/starter-kit-bot-details.md:1]: if you're referring to the `bot details` link, yep it works. I tested it locally.

File: docs/starter-get-started.md (hunk 3 above)
Comment (mwakaba2) [reply docs/starter-get-started.md:44]: sounds good, updated.

File: docs/starter-get-started.md (hunk 2 above)
Comment (mwakaba2) [reply docs/starter-get-started.md:5]: makes sense, updated.
